cdap-e2e-tests/src/e2e-test/resources/pluginDataCyAttributes.properties

```diff
@@ -0,0 +1,10 @@
+systemPreferences=system-prefs-accordion
+keyValue=key-value-pair-
+existingDataProc=provisioner-gcp-existing-dataproc
+gcpDataProc=provisioner-gcp-dataproc
+remoteHadoopProvisioner=provisioner-remote-hadoop
+Create=profile-create-btn
+Finish=wizard-finish-btn
```

Comment:
can we use key with starting word small? like create, finish, next, previous

Reply:
Yes for NameSpace admin only keyValue key is used as of now , will have to change in surya's commit as most of this attributes are used for SysAdmin.

Reply:
Done .